src/pvrclient-nextpvr.cpp

```diff
@@ -2111,7 +2104,10 @@ PVR_ERROR cPVRClientNextPVR::AddTimer(const PVR_TIMER &timerinfo)
   std::string encodedName = UriEncode(timerinfo.strTitle);
   std::string encodedKeyword = UriEncode(timerinfo.strEpgSearchString);
   std::string days = GetDayString(timerinfo.iWeekdays);
-  std:string directory = UriEncode( m_recordingDirectories[timerinfo.iRecordingGroup]);
+  std::string directory = UriEncode( m_recordingDirectories[timerinfo.iRecordingGroup]);
```

Comment:
These could be const if you want. Up to you

Reply:
Right, I did say I would change ares of code that I was touching so I will update those